build/configs/imxrt1020-evk/imxrt1020-evk_download.sh

```diff
@@ -18,64 +18,21 @@
 ###########################################################################
 # imxrt1020-evk_download.sh
 
-WARNING="\n########################################\n
-	WARNING:Make sure NO other application like \n
-	putty etc, is accessing ttyACM0. If download \n
-	fails, reset the board before next try!!\n
-	#################################################\n"
-
-echo -e $WARNING
-
-CURDIR=$(readlink -f "$0")
-CURDIR_PATH=$(dirname "$CURDIR")
-
-# When location of this script is changed, only TOP_PATH should be changed together!!!
-TOP_PATH=${CURDIR_PATH}/../../..
-
-USBRULE_PATH=${CURDIR_PATH}/../usbrule.sh
+USBRULE_PATH=${CUR_PATH}/usbrule.sh
 USBRULE_BOARD="imxrt"
 USBRULE_IDVENDOR="0d28"
 USBRULE_IDPRODUCT="0204"
 USBRULE_IDVENDOR2="0403"
 USBRULE_IDPRODUCT2="6010"
 
-OS_PATH=${TOP_PATH}/os
-OUTBIN_PATH=${TOP_PATH}/build/output/bin
-TTYDEV="/dev/ttyACM0"
-TINYARA_BIN=${OUTBIN_PATH}/tinyara.bin
-CONFIG=${OS_PATH}/.config
-ZONEINFO=${OUTBIN_PATH}/zoneinfo.img
-FLASH_START_ADDR=0x60000000
-
-##Utility function for sanity check##
-function imxrt1020_sanity_check()
-{
-	if [ ! -f ${CONFIG} ];then
-		echo "No .config file"
-		exit 1
-	fi
-
-	source ${CONFIG}
-	if [[ "${CONFIG_ARCH_BOARD_IMXRT1020_EVK}" != "y" ]];then
-		echo "Target is NOT IMXRT1020_EVK"
-		exit 1
-	fi
-
-	if [ ! -f ${TINYARA_BIN} ]; then
-		echo "missing file ${TINYARA_BIN}"
-		exit 1
-	fi
+BOARD=IMXRT1020_EVK
```

Comment:
I want to split setting values and running codes. How about moving above into metafile?

Reply:
Done.